Tests/XcodeGenKitTests/SourceGeneratorTests.swift

```diff
@@ -400,12 +403,15 @@ class SourceGeneratorTests: XCTestCase {
                     - B:
                       - b.swift
                       - c.jpg
+                    - D2.0:
```

Comment:
Can you also include all of the `directoryExtensionsToSkip` in a test somewhere, to ensure we didn't break those?

Reply:
- There is already a separate test for `xcdatamodel` [here](https://github.com/yonaskolb/XcodeGen/blob/b1edbda5831ae8f0a996ef7aec5112bc1195be51/Tests/XcodeGenKitTests/SourceGeneratorTests.swift#L101)
- Existing tests for `lproj` [here](https://github.com/yonaskolb/XcodeGen/blob/b1edbda5831ae8f0a996ef7aec5112bc1195be51/Tests/XcodeGenKitTests/SourceGeneratorTests.swift#L166) and [here](https://github.com/yonaskolb/XcodeGen/blob/b1edbda5831ae8f0a996ef7aec5112bc1195be51/Tests/XcodeGenKitTests/SourceGeneratorTests.swift#L124)
- tests for `xcassets` were already covered in the same test linked to this comment
- I have updated the test to include an assert for `bundle` though